Epam.TestAutomation/Epam.TestAutomation.BDD/DataDrivenTestingTests/Locators.cs

```diff
@@ -0,0 +1,26 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epam.TestAutomation.BDD
+{
+    public static class Locators
+    {
```

Comment:
It is not a good practice to put all locators in one class. Please move appropriate locators into page objects. Imagine that you have 10000 elements and all of them in 1 class. It would be impossible to find required one